python/ray/data/read_api.py

```diff
@@ -1040,11 +1040,10 @@ def read_json(
 
         >>> import pyarrow.json as pajson
         >>> block_size = 10 << 20 # Set block size to 10MB
-        >>> ray.data.read_json(
+        >>> ray.data.read_json( # doctest: +SKIP
```

Comment:
Nit: by convention, add two spaces
```suggestion
        >>> ray.data.read_json(  # doctest: +SKIP
```